tests/ui/suggestions/derive-macro-missing-bounds.stderr

```diff
@@ -4,9 +4,8 @@ error[E0277]: `a::Inner<T>` doesn't implement `Debug`
 LL |     #[derive(Debug)]
    |              ----- in this derive macro expansion
 LL |     struct Outer<T>(Inner<T>);
-   |                     ^^^^^^^^ `a::Inner<T>` cannot be formatted using `{:?}`
+   |                     ^^^^^^^^ the trait `Debug` is not implemented for `a::Inner<T>`
```

Comment:
I'm neutral about this change. Maybe some beginners need to know the relationship between Debug and `{:?}`. I am not sure about this.